snuba/clusters/storage_sets.py

```diff
@@ -0,0 +1,30 @@
+from typing import Mapping, Set
+
+from enum import Enum
+from snuba.datasets.storages import StorageKey
+
+
+class StorageSetKey(Enum):
+    EVENTS = "events"
+    GROUPASSIGNEES = "groupassignees"
```

Comment:
groupassignees is supposed to be joined with events so it should be in the same storage set.